static/js/global-nav.js

```diff
@@ -0,0 +1 @@
+../../node_modules/global-nav/dist/index.js
```

Comment:
A better solution is to update the npm script that builds the site to `cp` this file instead of using a symlink